engine/execution/computation/computer/result_collector.go

```diff
@@ -274,6 +279,14 @@ func (collector *resultCollector) processTransactionResult(
 		return nil
 	}
 
+	// TODO(ramtin): trigger consumers concurrently
```

Comment:
result collector is already pipeline.  most of the time, it's stall waiting for transaction results. we probably won't benefit much from additional concurrency (except for the last collection)

Reply:
from TPS perspective yes, but might be consumer latency they could benefit from faster calls, e.g. provider or bitswap make things available to the outside world faster and not waiting for other consumers

Reply:
removed the comment for now 
[066683c](https://github.com/onflow/flow-go/pull/4049/commits/066683c63c4de690855a8e6443d9a51c769e12dc)